pkg/utils/utils.go

```diff
@@ -303,7 +302,7 @@ func InitResourceMap(conf *config.Config) {
 		AllowedKubectlVerbMap[r] = true
 	}
 
-	resourceList, err := DiscoveryClient.ServerResources()
+	_, resourceList, err := DiscoveryClient.ServerGroupsAndResources()
```

Comment:
`ServerResources` was deprecated and removed in 1.24. For example, see comment from 1.22:
https://github.com/kubernetes/client-go/blob/v0.22.7/discovery/discovery_client.go#L98